File: samples/Mathematica/set.m
set1 = {"a", "b", "c", "d", "e"}; set2 = {"a", "b", "c", "d", "e", "f", "g"};
MemberQ[set1, "a"]
Union[set1 , set2]
Intersection[set1 , set2]
Complement[set2, set1](*Set Difference*)
MemberQ[Subsets[set2], set1](*Subset*)
set1 == set2(*Equality*)
set1 == set1(*Equality*)

File: samples/Clojure/set.clj
(require 'clojure.set)

; sets can be created using the set method or set literal syntax
(def a (set [1 2 3 4]))
(def b #{4 5 6 7})

(a 10) ; returns the element if it's contained in the set, otherwise nil

(clojure.set/union a b)

(clojure.set/intersection a b)

(clojure.set/difference a b)

(clojure.set/subset? a b)
